src/main/java/de/samply/share/broker/rest/Searchbroker.java

```diff
@@ -240,108 +240,6 @@ public Response getAllDirectoryIdOptions() {
     }
   }
 
-  /**
```

Comment:
is this related to the feature?

Reply:
No, just wanted to delte unnecessary code